Demo/Demo/Models/SetupTokenResponse.swift

```diff
@@ -1,6 +1,16 @@
 import Foundation
 
-struct SetUpTokenResponse: Decodable {
+struct SetUpTokenResponse: Decodable, Equatable {
+
+    static func == (lhs: SetUpTokenResponse, rhs: SetUpTokenResponse) -> Bool {
+        lhs.id == rhs.id
+    }
```

Comment:
Can we move this under the properties and add a `// MARK: - Equatable Conformance` above it?

Reply:
Oops, merged before seeing this. I will put it on the next PR.